src/domain/service/applications/index.ts

```diff
@@ -90,7 +90,7 @@ export async function create(identity: UserIdentity) {
  * applications remain linked in that scenario
  * ```
  * @param renewalApp Renewal Application that has already been updated in state
- * @param sourceAppId
+ * @param sourceAppId sourceAppId from original renewal app object
```

Comment:
> Definition, noun:
> Definition of a word.

Reply:
i think i'll add `sourceAppId` one more time for spite